README.md

```diff
@@ -1,2 +1,4 @@
 # python-class
 Projects I did in class of 2023
+
+Here is the note that your father added. We later worked on some [PyGameZero](https://pygame-zero.readthedocs.io/en/stable/) projects.
```

Comment:
do we need to say who added this?

Reply:
Okay thank you I've made a commit with your consideration

Reply:
thanks